### DISCUSSION TITLE
Quickstart reference to customizing outputs

### DISCUSSION THREAD

**Discussion by mahf708** [E3SM-Project/zppy#407] [Questions] [answered]
We are interested in customizing a few outputs. For example, it could be that `zppy -c ...cfg` produces `X` and `Y` separately and plot them nicely. We are in turn interested in something like `5*X + Y` to be plotted in the same manner. I realize this is may be asking too much of a nifty utility like `zppy`, but who knows?! It might already exist :)

Is there a quickstart reference for general customizability of zppy? Thanks!

**Comment by forsyth2** [comment, marked as answer]
I'm not exactly sure what the request is. Customizing the plots produced by `e3sm_diags` or `global_time_series`? Each of those have some parameters that allow for customization. Otherwise, a change in `zppy`'s source code would be required. 

Note, however, that #402 will add plugin capability for user-created ad-hoc (i.e., not intended for a wide audience or long-term use) tools.

**Reply by mahf708** [reply]
Is customization allowed on the level of arithmetics? I am mostly concerned with e3sm_diags

**Reply by forsyth2** [reply]
@chengzhuzhang may be able to say more about E3SM Diags specifically, but I think anything at that level would necessarily require a source code change.

**Reply by mahf708** [reply]
Yeah, that's what I thought. This is not a "request", more like an "informational question"  ... this capability goes beyond what I would expect maintainers to support, so I am going to mark this as answered and I will post separate questions about specific plotting questions later. Thanks a lot for the speedy response, as always :D
